src/main/java/Deadline.java

```diff
@@ -0,0 +1,25 @@
+public class Deadline extends Task{
+    private static final String ALPHABET_D = "D";
+    private String by;
+
+    public Deadline(String d, String by){
+        super(d);
+        this.by = by;
+    }
+
+    @Override
```

Comment:
I like the use of Override for methods, as they seem to be used appropriately. (I have noticed this for several other methods as well)